src/main/java/com/notably/model/viewstate/ViewStateModel.java

```diff
@@ -1,7 +1,30 @@
 package com.notably.model.viewstate;
 
+import java.util.Optional;
+
+import javafx.beans.property.Property;
+
 /**
  * API for the ViewStateModel component.
  */
 public interface ViewStateModel extends CommandInputModel, HelpFlagModel {
+
+    /**
+     * Shows user the meaning of their input, e.g. "Create a new note".
+     * When input cannot get corrected, the "error" message will be shown in commandTextProperty.
```

Comment:
Ooops a bit of typo here. Should be `responseTextProperty`

Reply:
oh yeas sorry forgot to change